src/views/Room.vue

```diff
@@ -202,6 +203,18 @@ export default {
         },
       })
     },
+    reconnectRtc() {
+      if (this.signalingConnectedBefore) {
+        // TODO: show "Palava server not reachable" or "Network not reachable" overlay
+        if (navigator.online) {
+          rtc.reconnect()
```

Comment:
Although we are guarding against being offline, the network can be bad (or the server could be down), so there should probably be a timeout functionality here. Maybe also a retry count (or an "endless" interval with larger timeout)

Reply:
And what happens after the retry count? The person gets kicked out? I think as long as somebody wants to participate in the discussion, it should be continously tried to reconnect?

Reply:
Fair point. One way would be to then show a network error page with a "reconnect" button. But the endless-interval with reasonable timeout in between tries sounds fine too me too.